ai/Gemini_ai_call_server.py

```
@@ -0,0 +1,846 @@
+import traceback
+import time
+import json
+from typing import Dict, List, Set
+
+import pyaudio
+from google import genai
+from google.genai import types
+from dotenv import load_dotenv
+import websockets
+import aiohttp  # aiohttp 모듈 추가
+from fastapi import FastAPI, WebSocket, WebSocketDisconnect, Query, status, HTTPException, Body, Header
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)